lib/makeWebpackConfig.js

```diff
@@ -171,7 +171,7 @@ module.exports = async (playroomConfig, options) => {
   };
 
   const theirConfig = playroomConfig.webpackConfig
-    ? playroomConfig.webpackConfig()
+    ? await Promise.resolve(playroomConfig.webpackConfig())
```

Comment:
Is the `Promise.resolve` call redundant here?

Reply:
I think maybe it is. Afaik you cant await something that isnt thenable? Or at least I've got eslint rules that complain when you're using await on non-thenables.

Reply:
It doesn't look like `webpack-merge` supports promises https://github.com/survivejs/webpack-merge/issues/81